### PR TITLE
Update API and UI

### PR DESCRIPTION
**Pull request by nyirit**
* Removes `xxxxx` from frontpage
* Turns off sign up
* Updates partners page
* Adds link for partners page in main menu
* Adds notification about new partner registrations
* Refactors partner registration submission

### CODE REVIEW

File: scoap3/modules/theme/templates/scoap3_theme/_navbar.html [hunk 1]
```diff
@@ -22,6 +22,7 @@
     <ul class="nav navbar-nav">
       <li><a href="/">Home</a></li>
       <li><a target="_blank" href="https://scoap3.org/">SCOAP<sup>3</sup> project</a></li>
+      <li><a href="/partners">For partners</a></li>
```
Comment (ammirate) [inline scoap3/modules/theme/templates/scoap3_theme/_navbar.html]: Only "Partners" would be more consistent IMO

File: tests/integration/api/test_registraion.py [hunk 2]
```diff
@@ -0,0 +1,56 @@
...
+    m = mock.MagicMock()
+    m.method = 'POST'
+    m.form = {
+        'email': email,
+    }
+
+    with mock.patch("scoap3.modules.api.views.request", m),\
+            mock.patch("scoap3.modules.api.views.flash"):
+        assert handler_registration() is True
+
```
Comment (ammirate) [inline tests/integration/api/test_registraion.py:39]: I would check that the user is actually in the DB as we expect

**Review by ammirate — COMMENTED** [review]
Looks good overall, just a couple of things

File: scoap3/modules/theme/templates/scoap3_theme/_navbar.html (hunk 1 above)
Comment (nyirit) [reply scoap3/modules/theme/templates/scoap3_theme/_navbar.html]: For me a Partners link would mean that there will be some information about the partners. But here we provide information useful **for** the partners and not about them.

**Review by ammirate — APPROVED** [review]
Approved :rocket:

**Pull request merged**